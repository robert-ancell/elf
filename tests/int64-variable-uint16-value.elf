uint16 v = 65535
int64 value = v
print (value)
