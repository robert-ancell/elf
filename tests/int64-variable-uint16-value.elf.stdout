Line 2:
int64 value = v
      ^^^^^
Variable is of type int64, but value is of type uint16
